crates/handlers/Cargo.toml

```diff
@@ -34,6 +34,7 @@ tower-http.workspace = true
 axum.workspace = true
 axum-macros = "0.4.1"
 axum-extra.workspace = true
+reqwest = { version = "0.12", default-features = false, features = ["json", "rustls-tls"] }
```

Comment:
So pulling `reqwest` might be something we want to do in the whole project. Right now we have a bit of a manual HTTP client based on `hyper` with a lot of metrics/tracing capabilities, and that use the right CA certificates & co.

I'll try to do a PR to move to reqwest overall, because it feels a lot simpler, but I would like to avoid having a mix of both reqwest and the current http client

Reply:
Yes it's true such change should actually be in its own PR and be global. So for now I should use the current `hyper` implementation (I must admit, I didn't manage to make it work in my first attempts, but I will try again ;-) ).